applications/nrf5340_audio/src/bluetooth/bt_content_control/media/bt_content_ctrl_media_internal.h

```diff
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-#ifndef _BT_MEDIA_CONTROL_H_
-#define _BT_MEDIA_CONTROL_H_
+#ifndef _BT_CONTENT_CTRL_MEDIA_H_
```

Comment:
```suggestion
#ifndef _BT_CONTENT_CTRL_MEDIA_INTERNAL_H_
```

I need to do the same, hehe

Reply:
Done